modules/sagemaker/sagemaker-templates-service-catalog/tests/test_stack.py

```diff
@@ -51,6 +53,12 @@ def stack(stack_defaults) -> cdk.Stack:
     prod_security_group_ids = ["sg"]
     sagemaker_domain_id = "domain_id"
     sagemaker_domain_arn = f"arn:aws:sagemaker:::domain/{sagemaker_domain_id}"
+    repository_type = RepositoryType.CODECOMMIT
+    access_token_secret_name = ("github_token",)
+    aws_codeconnection_arn = (
+        "arn:aws:codeconnections:xxxxxx:xxxxxxxxxxxx:connection/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+    )
```

Comment:
Are these two above supposed to be tuples as well?

Reply:
They aren't tuples! Fixed!